### ISSUE TITLE
bug: EXC_BREAKPOINT in registerPlugins

### ISSUE THREAD

**Issue by Sasha4ever** [ionic-team/capacitor#2532]
#2532  Bug Report

## Capacitor Version
`npx cap doctor` output:
💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 1.5.1

  @capacitor/core: 1.5.1

  @capacitor/android: 1.5.1

  @capacitor/ios: 1.5.1

Installed Dependencies:

  @capacitor/android not installed

  @capacitor/cli 1.5.1

  @capacitor/core 1.5.1

  @capacitor/ios 1.5.1

  Found 0 Capacitor plugins for ios:
[success] iOS looking great! 👌

## Affected Platform(s)

- [ ] Android
- [x] iOS
- [ ] Electron
- [ ] Web

## Current Behavior
run app on device. And registerPlugins crash

## Reproduction Steps

build app on device. (on simulator all ok)

## Other Technical Details

`npm --version` output: 6.13.7

`node --version` output: 10.17.0
`pod --version` output (iOS issues only): 1.8.4
xcode: 11.4b2

## Other Information

po classes![0]
2020-03-06 13:02:41.565498+0500 XXXXX[11127:3805099] *** NSForwarding: warning: object 0x2027c4948 of class 'PFEmbeddedMulticasterImplementation' does not implement methodSignatureForSelector: -- trouble ahead
2020-03-06 13:02:41.565830+0500 XXXXX[11127:3805099] *** NSForwarding: warning: object 0x2027c4948 of class 'PFEmbeddedMulticasterImplementation' does not implement doesNotRecognizeSelector: -- abort
error: Execution was interrupted, reason: EXC_BREAKPOINT (code=1, subcode=0x1b802b438).
The process has been returned to the state before expression evaluation.

<img width="921" alt="xcode" src="https://user-images.githubusercontent.com/1776959/76065264-1c7a9d00-5fad-11ea-8111-eec7da197a68.png">

**Comment by Sasha4ever** [comment]
I will try https://github.com/ionic-team/capacitor/pull/2414

**Comment by jcesarmobile** [comment]
have you tried with xcode 11.4 beta 3?

**Comment by Sasha4ever** [comment]
yes, on b3 still crashing

**Comment by jcesarmobile** [comment]
I've merged https://github.com/ionic-team/capacitor/pull/2414, closing

**Comment by aaronksaunders** [comment]
what is the fix? I am still seeing this error...

**Comment by shanoinsano10** [comment]
Also still seeing the error.

**Comment by jcesarmobile** [comment]
use `@capacitor/ios@next`

**Comment by davidseek** [comment]
> @capacitor/ios@next

That fixed it for me: `npm i @capacitor/ios@next`

**Comment by SSuarezBrooktec** [comment]
Same here

**Comment by ryandegruyter** [comment]
Does not fix it for me, even after npm i @capacitor/ios@next

Any other steps I should take after npm i @capacitor/ios@next?

**Comment by adonus19** [comment]
I am experiencing the same. I ran `npm i @capacitor/ios@next` and still have the same issue as before. I also updated `@capacitor/core`. Both packages are now at version 2.0.0-beta.1. I uninstalled and reinstalled Xcode as well. 

What am I missing?

**Comment by aaronksaunders** [comment]
did you resync / rebuild afterwards?

--

Aaron K. Saunders
CEO Clearly Innovative Inc - Luma Lab - In3
aaron@clearlyinnovative.com
www.clearlyinnovative.com
www.In3DC.com



This email message and any attachment(s) are for the sole use of the
intended recipient(s) and may contain proprietary and/or confidential
information which may be privileged or otherwise protected from disclosure.
Any unauthorized review, use, disclosure or distribution is prohibited. If
you are not the intended recipient(s), please contact the sender by reply
email and destroy the original message and any copies of the message as
well as any attachment(s) to the original message.

**Comment by adonus19** [comment]
Multiple times. I ran `ionic capacitor run ios` to handle building and syncing. Is there another command I should try?

**Comment by adonus19** [comment]
By using this page [here](https://capacitor.ionicframework.com/docs/ios/updating) I was able to resolve this issue. However, I am now encountering a new one. 

`dyld: Library not loaded: @rpath/protobuf.framework/protobuf`
`Reason: image not found`

I'm seeing that other have faced a similar issue when using a free Apple Developer account. However, I am using a paid account. But, this is another issue.

**Comment by SSuarezBrooktec** [comment]
try 

`ionic cap update ios`
or

`ionic cap sync ios`

**Comment by bart-blommaerts** [comment]
This did it for me:

> Update native project to Swift 5

Capacitor 2.0 uses Swift 5, it's recommended to update your native project to also use Swift 5. To do so, from Xcode click Edit -> Convert -> To Current Swift Syntax.

App.app will appear selected, click Next button.

Then a message will say No source changes necessary.

Finally, click the Update button.

**Comment by cuni0716** [comment]
> I've merged #2414, closing

I can see this error in apps that are working some weeks ago, I think that this issue isn't closed yet...

**Comment by bhandaribhumin** [comment]
still not work, I tried every possible for fix this issue but still same error I'm getting


```
2020-07-13 17:48:53.412203+0530 App[3990:806089] libMobileGestalt MobileGestaltCache.c:166: Cache loaded with 4563 pre-cached in CacheData and 45 items in CacheExtra.
2020-07-13 17:48:54.323138+0530 App[3990:805921] *** NSForwarding: warning: object 0x1fd944ef0 of class 'PFEmbeddedMulticasterImplementation' does not implement methodSignatureForSelector: -- trouble ahead
2020-07-13 17:48:54.323267+0530 App[3990:805921] *** NSForwarding: warning: object 0x1fd944ef0 of class 'PFEmbeddedMulticasterImplementation' does not implement doesNotRecognizeSelector: -- abort
(lldb) 
```
